packages/react-instantsearch/src/components/List.js

```diff
@@ -80,14 +80,14 @@ class List extends Component {
   renderShowMore() {
     const {showMore, translate, applyTheme} = this.props;
     const {extended} = this.state;
-
+    const disabled = this.props.limitMin >= this.props.items.length;
     if (!showMore) {
       return null;
     }
 
     return (
       <button
-        {...applyTheme('showMore', 'showMore')}
+        {...applyTheme('showMore', 'showMore', disabled && 'showMoreDisabled')}
```

Comment:
I tried to stay consistent with our current way to apply conditional theme key. You'll get this in the dom: 
```html
<button class="ais-RefinementList__showMore ais-RefinementList__showMoreDisabled" disabled>Show more</button>
<!-- or -->
<button class="ais-RefinementList__showMore">Show more</button>
```
If we change the way to compute theme here, It would be better to do it everywhere no? And since I believe that we will remove soon the react-themeable part, we would need a new way to compute classnames so maybe it's better to wait?